### PR TITLE
THF-214: news quote

### PR DESCRIPTION
**Pull request by AlirezaKes**
Foundation for Quote component

### CODE REVIEW

File: pages/[...slug].tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ import { Container } from 'hds-react'
 import NodeBasicPage from '@/components/pageTemplates/NodeBasicPage'
 import NodeLandingPage from '@/components/pageTemplates/NodeLandingPage'
 import NodeEventPage from '@/components/pageTemplates/NodeEventPage'
+import NodeAreticlePage from '@/components/pageTemplates/NodeArticlePage'
```
Comment (Arskiainen) [inline pages/[...slug].tsx:21]: Areticle?

**Review by Arskiainen — APPROVED** [review]
This looks alright.  I feel like i reviewed one of this already...

**Comment by Arskiainen** [comment]
You should merge this branch to the article branch, cause otherwise the squash merges are going to be painful.

**Review by Jussiles — APPROVED** [review]
Requested one change in Slack, otherwise looks goodio

**Pull request merged**